.github/actions/send-telegram-notify/action.yml

```diff
@@ -28,6 +28,12 @@ runs:
         uname -s | tee -a $GITHUB_ENV
         echo 'EOF' >> $GITHUB_ENV
       shell: bash
+    # check if host OS is rhel or fedora
+    - run: |
+        echo 'OS_IS_FEDORA_OR_RHEL<<EOF' >> $GITHUB_ENV
+        grep "^ID_LIKE=" /etc/os-release | grep -e 'rhel' -e 'fedora' && echo true | tee -a $GITHUB_ENV
```

Comment:
Firstly, you made this change according to CentOS 7 which ID_LIKE="rhel fedora". Why did you expand it for ID_LIKE="rhel" or ID_LIKE="fedora"? 
Secondly, why do not we need to check here `ID` field too? I mean, are you confident about there are no cases where, for instance, `ID="fedora"` but `fedora or rhel not in  ID_LIKE`

Reply:
You are right, ID_LIKE is not found in fedora of any versions. So I'll use 'ID' field which is common for CentOS and Fedora:
```
for img in el-7 el-8 fedora-28 fedora-29 fedora-30 fedora-31 fedora-32 fedora-33 ; do echo =========== $img ; docker run --rm -ti packpack/packpack:$img /bin/bash -c 'cat /etc/os-release' | grep ID ; done
=========== el-7
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
=========== el-8
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
=========== fedora-28
ID=fedora
VERSION_ID=28
PLATFORM_ID="platform:f28"
=========== fedora-29
ID=fedora
VERSION_ID=29
PLATFORM_ID="platform:f29"
VARIANT_ID=container
=========== fedora-30
ID=fedora
VERSION_ID=30
PLATFORM_ID="platform:f30"
VARIANT_ID=container
=========== fedora-31
ID=fedora
VERSION_ID=31
PLATFORM_ID="platform:f31"
VARIANT_ID=container
=========== fedora-32
ID=fedora
VERSION_ID=32
PLATFORM_ID="platform:f32"
VARIANT_ID=container
=========== fedora-33
ID=fedora
VERSION_ID=33
PLATFORM_ID="platform:f33"
VARIANT_ID=container
```